app/controllers/nodes_controller.rb

```diff
@@ -18,18 +18,34 @@ def edit
   # POST /nodes
   def create
     @node.label = 'unnamed' unless @node.label.present?
-    if @node.save
-      track_created(@node)
-      flash[:notice] = 'Successfully created node.'
-      redirect_to @node
-    else
-      parent = @node.parent
-      if parent
-        redirect_to parent, alert: @node.errors.full_messages.join('; ')
-      else
-        redirect_to summary_path, alert: @node.errors.full_messages.join('; ')
+    @node.save!
```

Comment:
True today, what if we add a new validation later? Isn't a standard `if .save else` quite a standard practice for controllers?